### PR TITLE
Add pdbonly to release configurations.

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

File: main/src/addins/WindowsPlatform/WindowsAPICodePack/Shell/Shell.csproj [hunk 1]
```diff
@@ -60,7 +62,6 @@
     <DefineConstants>TRACE;DEBUG;CODE_ANALYSIS</DefineConstants>
     <DocumentationFile>bin\Debug\Microsoft.WindowsAPICodePack.Shell.XML</DocumentationFile>
     <DebugType>full</DebugType>
-    <PlatformTarget>AnyCPU</PlatformTarget>
```
Comment (Therzok) [inline main/src/addins/WindowsPlatform/WindowsAPICodePack/Shell/Shell.csproj]: This is defined above.

File: main/src/addins/NUnit/MonoDevelop.NUnit.csproj [hunk 2]
```diff
@@ -174,7 +174,7 @@
     <EmbeddedResource Include="Gui\icons\project-nunit-overlay-32.png">
       <LogicalName>project-nunit-overlay-light-32.png</LogicalName>
     </EmbeddedResource><!-- TODO: VV: Duped definition to not break fsharpbinding (https://github.com/fsharp/fsharpbinding/blob/master/monodevelop/MonoDevelop.FSharpBinding/Templates/FSharpNUnitLibraryProject.xpt.xml#L8) -->
-    <EmbeddedResource Include="Gui\icons\project-nunit-overlay-32%402x.png">
+    <EmbeddedResource Include="Gui\icons\project-nunit-overlay-32@2x.png">
```
Comment (Therzok) [inline main/src/addins/NUnit/MonoDevelop.NUnit.csproj]: This shouldn't have landed like this. We probably overlooked it. I think we should stop XS from encoding characters.

File: main/src/addins/MonoDevelop.Refactoring/MonoDevelop.Refactoring.csproj [hunk 3]
```diff
@@ -34,7 +33,6 @@
     <DebugSymbols>true</DebugSymbols>
     <NoWarn>1591;1573</NoWarn>
     <DocumentationFile>..\..\..\build\AddIns\MonoDevelop.Refactoring\MonoDevelop.Refactoring.xml</DocumentationFile>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Refactoring/MonoDevelop.Refactoring.csproj]: Defined above.

File: main/src/addins/VersionControl/MonoDevelop.VersionControl.Subversion.Unix/MonoDevelop.VersionControl.Subversion.Unix.csproj [hunk 4]
```diff
@@ -50,10 +50,13 @@
     <DocumentationFile>..\..\..\..\build\AddIns\VersionControl\MonoDevelop.VersionControl.Subversion.Unix.xml</DocumentationFile>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'ReleaseMac|AnyCPU' ">
-    <Optimize>false</Optimize>
+    <Optimize>true</Optimize>
     <OutputPath>bin\ReleaseMac</OutputPath>
```
Comment (slluis) [inline main/src/addins/VersionControl/MonoDevelop.VersionControl.Subversion.Unix/MonoDevelop.VersionControl.Subversion.Unix.csproj]: Output path should be ..\..\..\..\build\AddIns\VersionControl

**Comment by slluis** [comment]
Ok to commit after the output path fix

File: main/src/addins/VersionControl/MonoDevelop.VersionControl.Subversion.Unix/MonoDevelop.VersionControl.Subversion.Unix.csproj (hunk 4 above)
Comment (Therzok) [inline main/src/addins/VersionControl/MonoDevelop.VersionControl.Subversion.Unix/MonoDevelop.VersionControl.Subversion.Unix.csproj]: Ah. Missed that. Will fix.

**Pull request merged**